src/Servers/IIS/IIS/src/Microsoft.AspNetCore.Server.IIS.csproj

```diff
@@ -26,7 +26,7 @@
 
   <ItemGroup Condition="'$(OS)' == 'Windows_NT' AND '$(VCTargetsPath)' != ''">
     <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj" Platform="x64" />
-    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj" Platform="x86" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj" Platform="Win32" />
```

Comment:
Why the change? We've been using x86 everywhere else. If we need to make this change, make sure _everything_ is updated to set Platform=Win32, not x86

Reply:
I'll make sure everything is Win32 in native project references.

Reply:
Native projects and IIS solution use Win32 platform, there was a fallback in default C++ targets that allowed things to function but it wasn't done correctly.